backend/src/main/resources/application.properties

```diff
@@ -30,13 +30,16 @@ quarkus.log.category."com.redhat.cloud.notifications".level=INFO
 %test.quarkus.http.access-log.category=info
 
 rbac.enabled=true
+internal-rbac.enabled=true
 
 # RBAC configuration used during user authentication. It is used when a public REST API is called.
 #rbac-authentication/mp-rest/url=http://ci.foo.redhat.com:1337
 rbac-authentication/mp-rest/url=${clowder.endpoints.rbac-service:https://ci.cloud.redhat.com}
 rbac-authentication/mp-rest/connectTimeout=2000
 rbac-authentication/mp-rest/readTimeout=2000
 
+internal-ui.admin-role=crc-notifications-team
```

Comment:
It shouldn't change - I just didn't felt OK harcoding it. The only reason I could see is that in the future someone else will manage it instead of us all directly.

Reply:
I would hardcode it but feel free to chose whatever you prefer :smiley: